tests/migrations/test_writer.py

```diff
@@ -17,8 +17,9 @@
 from django.db import migrations, models
 from django.db.migrations.serializer import BaseSerializer
 from django.db.migrations.writer import (
-    MigrationWriter, OperationWriter, SettingsReference,
+    MigrationWriter, OperationWriter,
 )
+from django.conf import SettingsReference
```

Comment:
Please ensure that imports are ordered alphabetically. In fact there is already a `from django.conf import ...` line above that this should be merged with. The `from django.db.migrations.writer import ...` statement can also be condensed onto one line without the parentheses.

Reply:
sorry about that. I always run isort before my pushs. I forgot this time.